cmd/helm/root.go

```diff
@@ -155,7 +155,7 @@ func newRootCmd(actionConfig *action.Configuration, out io.Writer, args []string
 	registryClient, err := registry.NewClient(
 		registry.ClientOptDebug(settings.Debug),
 		registry.ClientOptEnableCache(true),
-		registry.ClientOptWriter(os.Stderr),
```

Comment:
you forgot at least helm install

Reply:
thanks! updated.

Reply:
and helm template too :)

Reply:
alright, that one was easy to miss.   it doesn't use `bindOutputFlag` :)